### ISSUE TITLE
Segfault when break in block(mrb_yield)

### ISSUE THREAD

**Issue by bggd** [mruby/mruby#2323]
``` c
mrb_value
foo(mrb_state *mrb, mrb_value self)
{
  mrb_value blk;

  mrb_get_args(mrb, "&", &blk);

  printf("start foo\n");

  mrb_yield_argv(mrb, blk, 0, NULL);

  printf("end foo\n");

  return mrb_nil_value();
}
```

main.rb:

``` ruby
foo do
  p "in block"
  break
end
p "end script"
```

```
$ ./bin/mruby main.rb
start foo
"in block"
"end script"
end foo
Segmentation fault (core dumped)
```

**Comment by dearblue** [comment]
This issue is resolved in 1.3.0 and later.

I did not confirm the commit unit build, but it was probably resolved by https://github.com/mruby/mruby/commit/d4d99dd6d7e1374af3e567b175035b36977337c4.

**Comment by bggd** [comment]
It's ok on latest mruby.

Thanks.
